environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/CompactRegionResponse.java

```diff
@@ -17,22 +17,22 @@
 public class CompactRegionResponse implements Serializable {
 
     @ApiModelProperty(EnvironmentModelDescription.REGIONS)
-    private Set<String> regions;
+    private Set<String> values;
```

Comment:
we talked about naming this field `names` so it will be consistent with the `location` object